test/docfx.Test/lib/JsonUtilityTest.cs

```diff
@@ -544,6 +544,23 @@ public void SyntaxErrorShouldBeThrownWithoutSchemaValidation(string json)
             Assert.Equal(ErrorLevel.Error, exception.Error.Level);
         }
 
+        [Fact]
+        public void OmitEmptyEnumerableValue()
+        {
+            var content = JsonUtility.Serialize(new EmptyEnumerable());
+            Assert.Equal("{\"a\":\"\"}", content);
+        }
+
+        public class EmptyEnumerable
+        {
+            public string A { get; set; } = "";
+            public List<string> B { get; set; } = new List<string>();
```

Comment:
`List` is enough? We are not testing all types implementing `IEnumerable` anyway.
Possible to add a property with nested List? which should be omitted as well